source/tests/PuppTests.test.js

```diff
@@ -27,36 +46,6 @@ describe('basic navigation for BJ', () => {
         expect(headerText).toBe('Create your login!');
     });
 
-    /*
-
-    it('Test2: Try to access another page Daily - Shows create your login ', async () => {
-        await page.goto(
-            'http://127.0.0.1:5501/source/DailyOverview/DailyOverview.html'
-        );
-        await page.waitForTimeout(300);
-        const url = await page.evaluate(() => location.href);
-        expect(url).toMatch('http://127.0.0.1:5501/source/Login/Login.html');
-    });
```

Comment:
i saw these tests in other versions as well. If they are deprecated, perhaps we need to discuss with others whether we should remove these